Sources/Sentry/Public/SentryOptions.h

```diff
@@ -414,6 +414,30 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, retain) NSArray *tracePropagationTargets;
 
+/**
+ * When enabled, the SDK captures HTTP Client errors. Default value is NO.
+ */
+@property (nonatomic, assign) BOOL enableCaptureFailedRequests;
+
+/**
+ * The SDK will only capture HTTP Client errors if the HTTP Response status code is within the
+ * defined range.
+ *
+ * Defaults to 500 - 599.
+ */
+@property (nonatomic, strong) NSArray<SentryHttpStatusCodeRange *> *failedRequestStatusCodes;
+
+/**
+ * An array of hosts or regexes that determines if HTTP Client errors will be automatically
+ * captured.
+ *
+ * This array can contain instances of NSString which should match the URL (using `contains`),
+ * and instances of NSRegularExpression, which will be used to check the whole URL.
+ *
+ * The default value automatically captures HTTP Client errors of all outgoing requests.
+ */
+@property (nonatomic, strong) NSArray *failedRequestTargets;
```

Comment:
`tracePropagationTargets` is `retain` for some reason, but @brustolin said it should be `strong`.
Its unclear if I should change `tracePropagationTargets` as well or not.

Reply:
I found out that retain and strong are pretty much the same thing.

Reply:
So do I keep `tracePropagationTargets` as it is or change it to keep the pattern? is `strong` or `retain` preferable?

Reply:
ok, will keep it as it is.